recipe/meta.yaml

```diff
@@ -1,41 +1,37 @@
 {% set name = "healpix_cxx" %}
-{% set version = "3.31" %}
-{% set sha256 = "ddf437442b6d5ae7d75c9afaafc4ec43921f903c976e25db3c5ed5185a181542" %}
+{% set version = "3.80" %}
+{% set sha256 = "8de4aabaa3d55e0c63a62c621bfd77298732c6b75782016217f5659143acbcf5" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: Healpix_{{ version }}_2016Aug26.tar.gz
-  url: https://downloads.sourceforge.net/project/healpix/Healpix_{{ version }}/Healpix_{{ version }}_2016Aug26.tar.gz
+  fn: Healpix_{{ version }}_2021Jun22.tar.gz
+  url: https://downloads.sourceforge.net/project/healpix/Healpix_{{ version }}/Healpix_{{ version }}_2021Jun22.tar.gz
   sha256: {{ sha256 }}
 
 build:
-  number: 1003
+  number: 0
   skip: true  # [win]
 
 requirements:
   build:
-    - autoconf
-    - {{ compiler('fortran') }}
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - make
   host:
     - cfitsio
   run:
     - cfitsio
 
 test:
   commands:
-    - test -f ${PREFIX}/lib/libhealpix_cxx.a
-    - test -f ${PREFIX}/lib/libc_utils.a
-    - test -f ${PREFIX}/lib/libcxxsupport.a
-    - test -f ${PREFIX}/lib/libfftpack.a
-    - test -f ${PREFIX}/lib/libsharp.a
```

Comment:
Do we want to put these libraries in another package for static builds or are they simply not built anymore?

Reply:
The `libc_utils.a`, `libcxxsupport.a` and `libfftpack.a` are no longer built. However, `libhealpix_cxx.a` and `libsharp.a` are still built, though I don't know if they are used (I am not a developer of this software).  @Areustle , @jasercion or @giacomov can you comment on this?

Reply:
Our policy is to remove static libs and package them separately. I think deleting all of them in this PR would be fine, but others should weigh in.

Reply:
Would it be okay if I raise this in an issue so this PR can be merged? As I have other packages that are dependent on this version of `healpix_cxx` that I hope to put into conda-forge?